apps/src/StudioApp.js

```diff
@@ -462,12 +475,11 @@ StudioApp.prototype.assetUrl_ = function (path) {
  * @param {boolean} shouldPlayOpeningAnimation True if an opening animation is
  *   to be played.
  */
-StudioApp.prototype.reset = function (shouldPlayOpeningAnimation) {
+StudioApp.prototype.reset = function (/*shouldPlayOpeningAnimation*/) {
   // TODO (bbuchanan): Look for comon reset logic we can pull here
   // Override in app subclass
 };
```

Comment:
The check for unused variables is useful, but it also complains about this function argument to a no-op function that we expect to override.  This seemed like a semi-appropriate way to annotate the unused argument, given how Javascript handles arguments.